Small fixes in Content Lifecycle section (#899)
SUSE/doc-susemanager: modules/administration/pages/content-lifecycle.adoc

Content Lifecycle Management allows you to select software channels as sources, adjust them as required for your environment, and thoroughly test them before installing onto your production systems.

While you cannot directly modify vendor channels, you can clone them and then modify the clones by adding or removing packages and custom patches.

> You can then assign these cloned channels to test systems to ensure they work as expected and, once all tests pass, promote them to production servers.

Reviewer: ```suggestion
You can then assign these cloned channels to test systems to ensure they work as expected.
Then, when all tests pass, you can promote them to production servers.
```
Avoid sentences too long.
And the SUSE Style Guide wants us to use "when" instead of "once" ;)
